extras/systemd/foreman.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Foreman
+Documentation=https://theforeman.org
+After=network.target remote-fs.target nss-lookup.target
+
+[Service]
+Type=simple
+User=$FOREMAN_USER
+TimeoutSec=300
+WorkingDirectory=/usr/share/foreman
+ExecStart=/usr/bin/scl enable tfm "rails server -e $FOREMAN_ENV -p $FOREMAN_PORT -b $FOREMAN_BIND"
+Environment=FOREMAN_ENV=production FOREMAN_PORT=3000 FOREMAN_BIND=0.0.0.0 FOREMAN_USER=foreman
```

Comment:
Should this be `RAILS_ENV`?

Reply:
It's passed in via `-e` so I think it's not needed